Readme.md

```diff
@@ -37,6 +37,26 @@ items = [1,2,3]
 Parallel.each( -> { items.pop || Parallel::Stop }) { |number| ... }
 ```
 
+Reduce will return results colleced from each worker. So you need to merge them on the end.
+```Ruby
+result = Parallel.reduce(['a','b','c','d','a','b','c','d']) do |result,x|
+  result ||= Set.new
+  result << x
+  result
+end
+result.compact.reduce(&:+)
```

Comment:
I am not absolutely sure I do understand your question but compact is only for a case when processing was not done in some process and it has nil in result. The part of question I do not understand is `'x' + 'p'` part, where to do it?